cpp-package/include/mxnet-cpp/kvstore.hpp

```diff
@@ -14,8 +14,8 @@
 #include "mxnet-cpp/kvstore.h"
 #include "mxnet-cpp/optimizer.h"
 
-#ifndef KVSTORE_HPP
-#define KVSTORE_HPP
+#ifndef CPP_PACKAGE_INCLUDE_MXNET_CPP_KVSTORE_HPP_
```

Comment:
These are lint fixes which captured problems in the original cpp-package headers